hanabi/templates/hanabi/hanabicontest_list.html

```diff
@@ -64,6 +64,9 @@ <h1>All Contests</h1>
               <a href="{{ contest.get_absolute_url }}">{{ contest.variant_name }}</a>
             </td>
             <td>Completed</td>
+          {% elif contest.is_upcoming %}
+            <td class="table-primary">Starts in {{ contest.start_date }}</td>
```

Comment:
Can we flip the colors so that the upcoming is colored by `warning` and the active is colored by `primary`? (Right now it's vice-versa.)